### PR TITLE
Change to `printOptions` and fix docstrings

### LINKED ISSUE #108: Deprecation warnings
## Description
Running ADflow causes a number of DeprecationWarnings.
```
/home/mdolabuser/repos/adflow/adflow/pyADflow.py:3623: DeprecationWarning: invalid escape sequence \*
  """This the main python gateway for producing forward mode jacobian
/home/mdolabuser/repos/adflow/adflow/pyADflow.py:3800: DeprecationWarning: invalid escape sequence \*
  """This the main python gateway for producing reverse mode jacobian
/home/mdolabuser/repos/baseclasses/baseclasses/BaseSolver.py:184: DeprecationWarning: printCurrentOptions is deprecated. Use printOptions instead.
  warnings.warn("printCurrentOptions is deprecated. Use printOptions instead.", DeprecationWarning)
```
The first two are probably related to the way the docstrings are set up. The last one is due to an API change in `baseclasses`, simply rename `printCurrentOptions` with `printOptions`.

### PR DESCRIPTION
**Pull request by eirikurj**
## Purpose
Closes #108. 
This PR fixes the docstrings and now uses the `printOptions` functions.

## Type of change
What types of change is it?
_Select the appropriate type(s) that describe this PR_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Documentation update
- [ ] Maintenance update
- [ ] Other (please describe)

## Testing
Explain the steps needed to test the new code to verify that it does indeed address the issue and produce the expected behavior.

## Checklist
_Put an `x` in the boxes that apply._

- [ ] I have run `flake8` and `black` to make sure the code adheres to PEP-8 and is consistently formatted
- [ ] I have run unit and regression tests which pass locally with my changes
- [ ] I have added new tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation

### CODE REVIEW

File: adflow/pyADflow.py [hunk 1]
```diff
@@ -3651,7 +3651,7 @@ def computeJacobianVectorProductFwd(self, xDvDot=None, xSDot=None, xVDot=None, w
         Returns
         -------
         dwdot, funcsdot, fDot : array, dict, array
-            One or more of the these are return depending on the \*Deriv flags
+            One or more of the these are return depending on the \\*Deriv flags
```
Comment (nwu63) [inline adflow/pyADflow.py:3654]: Why do we need to escape the `*` character? Does it work without any slashes?

Comment (eirikurj) [reply adflow/pyADflow.py:3654]: The `*` is an inline emphasis character which then starts but does not end. It will create an undesired behavior.

Comment (nwu63) [reply adflow/pyADflow.py:3654]: Ah I see. In that case would it make more sense to surround it with backticks? i.e. ``*Deriv``

**Review by joanibal — APPROVED** [review]

File: adflow/pyADflow.py (hunk 1 above)
Comment (eirikurj) [reply adflow/pyADflow.py:3654]: Sure, that would work if we want this to be literal. Actually many other things in the docstring there should be done this way too. I will update this.

**Review by nwu63 — APPROVED** [review]

**Pull request merged**